Documentation/ExtensionArchitecture/Tutorials/Tea/Model.rst

```diff
@@ -0,0 +1,298 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Model
+..  _extbase_tutorial_tea_model:
+
+===================
+Model: a bag of tea
+===================
+
+The model in this example is kept quite basic: Each tea can have a title,
+a description and optionally an image attached.
+
+..  uml::
+
+    class Tea {
+        title
+        ==
+        description
+        ==
+        image
+    }
+
+The title and description are strings, the image is stored as relation
+to the class :php:`\TYPO3\CMS\Extbase\Domain\Model\FileReference` provided
+by Extbase as model.
+
+
+..  _extbase_tutorial_tea_model_database:
+
+The database model
+==================
+
+Let us translate this into SQL and store the scheme in a file called
```

Comment:
A schema is an organizational pattern, while a scheme is more a long the lines of an intrigue. ;-)

```suggestion
Let us translate this into SQL and store the schema in a file called
```